INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /job/MyProject/doDelete HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 132
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
INFO     core.jenkins_utils:jenkins_utils.py:201 deleting token with uuid eec0d4fd-bfe1-44b4-8c50-eb19238af724
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/descriptorByName/jenkins.security.ApiTokenProperty/revoke HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
INFO     tests.api.support.base_api:base_api.py:52 Generating Jenkins API token at: "/generateNewToken"
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/descriptorByName/jenkins.security.ApiTokenProperty/generateNewToken HTTP/1.1" 200 168
INFO     tests.api.support.base_api:base_api.py:63 Generated token: 1140...
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /createItem?name=MyProject HTTP/1.1" 200 0
INFO     tests.api.support.jenkins_client:jenkins_client.py:23 [POST /createItem?name=MyProject] Status: 200 - OK
DEBUG    conftest:conftest.py:90 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:90 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:90 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:90 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:90 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:90 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 140.0.7339.207 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 140.0.7339.207 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 140.0.7339.207
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 140.0.7339.207
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 5568 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:48791
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "GET /session/9c89c95ccbf13935f1d01c08f2e13d9e/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element/f.A4B5209D12F3AF4889F864A254A7AB92.d.E4AADC4DB52CD215BEB726C223E2683F.e.1/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element/f.A4B5209D12F3AF4889F864A254A7AB92.d.E4AADC4DB52CD215BEB726C223E2683F.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element/f.A4B5209D12F3AF4889F864A254A7AB92.d.E4AADC4DB52CD215BEB726C223E2683F.e.8/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "GET /session/9c89c95ccbf13935f1d01c08f2e13d9e/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "GET /session/9c89c95ccbf13935f1d01c08f2e13d9e/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "GET /session/9c89c95ccbf13935f1d01c08f2e13d9e/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: c3521430548801d60ce4f77e9e51bde3bb9e8446c81982cf3886160338c0bdb3
DEBUG    LoginPage:login_page.py:32 login crumb: c3521430548801d60ce4f77e9e51bde3bb9e8446c81982cf3886160338c0bdb3
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('xpath', "//table[@id='projectstatus']//tbody//td[3]/a"),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "GET /session/9c89c95ccbf13935f1d01c08f2e13d9e/element/f.A4B5209D12F3AF4889F864A254A7AB92.d.6F52BFD758008DA375AE856B1E7F1B88.e.65/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element/f.A4B5209D12F3AF4889F864A254A7AB92.d.6F52BFD758008DA375AE856B1E7F1B88.e.65/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "GET /session/9c89c95ccbf13935f1d01c08f2e13d9e/url HTTP/1.1" 200 0
DEBUG    MultiConfigProjectPage:ui_element.py:70 Click on locator ('xpath', "//span[text()='Configure']/..")
DEBUG    MultiConfigProjectPage:ui_element.py:28 waiting for element_to_be_clickable(('xpath', "//span[text()='Configure']/.."),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "GET /session/9c89c95ccbf13935f1d01c08f2e13d9e/element/f.A4B5209D12F3AF4889F864A254A7AB92.d.FB9CAA35189815D61B1FE11E05F8AF8D.e.85/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element/f.A4B5209D12F3AF4889F864A254A7AB92.d.FB9CAA35189815D61B1FE11E05F8AF8D.e.85/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "GET /session/9c89c95ccbf13935f1d01c08f2e13d9e/url HTTP/1.1" 200 0
DEBUG    MultiConfigProjectConfigPage:ui_element.py:70 Click on locator ('css selector', "button[data-section-id='source-code-management']")
DEBUG    MultiConfigProjectConfigPage:ui_element.py:28 waiting for element_to_be_clickable(('css selector', "button[data-section-id='source-code-management']"),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "GET /session/9c89c95ccbf13935f1d01c08f2e13d9e/element/f.A4B5209D12F3AF4889F864A254A7AB92.d.6CB0972895D39EDA668081C5A416540D.e.257/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element/f.A4B5209D12F3AF4889F864A254A7AB92.d.6CB0972895D39EDA668081C5A416540D.e.257/click HTTP/1.1" 200 0
DEBUG    MultiConfigProjectConfigPage:ui_element.py:28 waiting for visibility_of_element_located(('css selector', "[name='userRemoteConfigs'] .validation-error-area--visible>.error"),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element HTTP/1.1" 404 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "POST /session/9c89c95ccbf13935f1d01c08f2e13d9e/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "GET /session/9c89c95ccbf13935f1d01c08f2e13d9e/element/f.A4B5209D12F3AF4889F864A254A7AB92.d.6CB0972895D39EDA668081C5A416540D.e.277/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48791 "DELETE /session/9c89c95ccbf13935f1d01c08f2e13d9e HTTP/1.1" 200 0